app/src/main/java/com/dsvag/currencyexchanger/data/until/OnItemClickListener.kt

```diff
@@ -0,0 +1,5 @@
+package com.dsvag.currencyexchanger.data.until
+
+interface OnItemClickListener {
```

Comment:
Можно дописать в начало `fun`, чтобы использовать как лямбды
```suggestion
fun interface OnItemClickListener {
```